src/game-logic/gameMap.js

```diff
@@ -0,0 +1,22 @@
+export default function gameMapCreator(mapSize) {
+  // object to store all the map tiles
+  // -- each tile will represent a cell. Each cell will have an x coord, y coord, and a status of whether or not it is alive
+  const gameMap = {};
```

Comment:
Very interesting approach!  I used an array for my board but I can definitely see the benefits of an object instead - awesome 🙌🏻